walletlibrary/src/main/java/com/microsoft/walletlibrary/util/http/httpagent/IHttpAgent.kt

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.walletlibrary.util.http.httpagent
+
+abstract class IHttpAgent {
+
+    open class HttpAgentError(val response: IResponse): Error() {}
```

Comment:
Why is this an error and not an exception?

Reply:
No particular reason; I don't think I was particularly aware of Kotlin having both.